### ISSUE TITLE
Collapse identical lines in the debug console

### ISSUE THREAD

**Issue by DanTup** [microsoft/vscode#104162]
In the Chrome dev tools, duplicate lines are collapsed together:

![Screenshot 2020-08-06 at 10 12 30](https://user-images.githubusercontent.com/1078012/89514184-6d492880-d7cd-11ea-8ce6-da2c90092fd7.png)

Here, the `console.log` has run 19 times but because the output was the same, it just shows a "19" alongside the first print.

I think this would work great in the Debug Console here too as it's not uncommon to have the same thing being printed a lot.

**Comment by isidorn** [comment]
This is a fair feature request, however we do not have it on our immediate plan. Thus assigning to backlog and we might add this in the future. Thanks!

**Comment by isidorn** [comment]
This is now supported, try it out with VS Code insiders from Tuesday and let us know how it works for you!

<img width="680" alt="Screenshot 2020-11-16 at 13 31 48" src="https://user-images.githubusercontent.com/1926584/99252941-1a413180-2810-11eb-8808-db77ba622f08.png">

**Comment by DanTup** [comment]
<img width="234" alt="Screenshot 2020-11-18 at 11 57 40" src="https://user-images.githubusercontent.com/1078012/99527790-4934e000-2995-11eb-820b-9c2221984612.png">

Works great (although maybe it could do with some padding between it and the text). Thanks!

**Comment by isidorn** [comment]
I went with 2 px pading. I have now increased it to 4px.
Based on your feedback adding verified label, thanks!

**Comment by collinmdp** [comment]
What in the world? How do I disable this feature? I just spent the last few hours printing bits to the debug console and parsing the output using a diff tool with what is expected. The last time I ran this test, this feature didn't exist and everything worked as expected. When I right click and select "Copy All" it copies duplicate lines only once, which leaves out big groupings of bytes. The copied output doesn't even tell you that anything has been removed from the output. I can't find anything in the VSCode preferences to remove this option.
